tests/utilities/test_apply_func_torchtext.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import torch
+import torchtext
+from torchtext.data.example import Example
+
+from pytorch_lightning.utilities.apply_func import move_data_to_device
+
+
+def _get_torchtext_data_iterator(include_lengths=False):
+    text_field = torchtext.data.Field(sequential=True, pad_first=False,  # nosec
+                                      init_token="<s>", eos_token="</s>",  # nosec
+                                      include_lengths=include_lengths)  # nosec
+
+    example1 = Example.fromdict({"text": "a b c a c"}, {"text": ("text", text_field)})
+    example2 = Example.fromdict({"text": "b c a a"}, {"text": ("text", text_field)})
+    example3 = Example.fromdict({"text": "c b a"}, {"text": ("text", text_field)})
+
+    dataset = torchtext.data.Dataset([example1, example2, example3],
+                                     {"text": text_field}
+                                     )
+    text_field.build_vocab(dataset)
+
+    iterator = torchtext.data.Iterator(dataset, batch_size=3,
+                                       sort_key=None, device=None,
+                                       batch_size_fn=None,
+                                       train=True, repeat=False, shuffle=None,
+                                       sort=None, sort_within_batch=None)
+    return iterator, text_field
+
+
+@pytest.mark.parametrize('include_lengths', [False, True])
+@pytest.mark.parametrize(['device'], [pytest.param(torch.device('cuda', 0))])
```

Comment:
```suggestion
@pytest.mark.parametrize(['device'], [pytest.param(torch.device('cpu')), pytest.param(torch.device('cuda', 0))])
```
let's see if it works, we can always undo the commit.

Reply:
ok